docs/components/SegmentedControl/Web.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { Meta, StoryFn } from "@storybook/react";
+import { SegmentedControl } from "@jobber/components/SegmentedControl";
+
+const meta: Meta = {
+  title: "Components/Selections/SegmentedControl/Web",
+  component: SegmentedControl,
+  parameters: {
+    viewMode: "story",
+    previewTabs: {
+      code: {
+        hidden: false,
+      },
+    },
+  },
+};
+export default meta;
+
+type Story = StoryFn<typeof SegmentedControl>;
+const options = [
+  { value: "pizza", label: "Pizza" },
+  { value: "tacos", label: "Tacos" },
+  { value: "sushi", label: "Sushi" },
+  { value: "burgers", label: "Burgers" },
+];
+
+export const BasicControlled: Story = () => {
+  const [activeOption, setActiveOption] = useState("pizza");
+
+  return (
+    <SegmentedControl
+      selectedValue={activeOption}
+      onSelectValue={setActiveOption}
+    >
+      {options.map(option => (
+        <SegmentedControl.Option key={option.value} value={option.value}>
+          {option.label}
+        </SegmentedControl.Option>
+      ))}
+    </SegmentedControl>
+  );
+};
+
```

Comment:
I also want to note that a more type safe implementation can be written, if that is preferred. 

```
type ControlledOptions = "pizza" | "pizza2" | "pizza3" | "pizza4";

export const Controlled: Story = () => {
  const [activeOption, setActiveOption] = useState<ControlledOptions>("pizza");

  return (
    <SegmentedControl<ControlledOptions>
      selectedValue={activeOption}
      onSelectValue={setActiveOption}
    >
      <SegmentedControl.Option<ControlledOptions> value="pizza">
        Pizza
      </SegmentedControl.Option>
      <SegmentedControl.Option<ControlledOptions> value="pizza2">
        Ultra Pizza
      </SegmentedControl.Option>
      <SegmentedControl.Option<ControlledOptions> value="pizza3">
        Mega Pizza
      </SegmentedControl.Option>
      <SegmentedControl.Option<ControlledOptions> value="pizza4">
        Giga Pizza
      </SegmentedControl.Option>
    </SegmentedControl>
  );
};
```

`SegmentedControl` and `SegmentedControlOption` can be used with the generic type parameter (`ControlledOptions`) which ensures type safety and restricts the `value` to a specific set of options.